lib/dirList.js

```diff
@@ -159,12 +160,9 @@ const dirList = {
    * @return {boolean}
    */
   handle: function (route, options) {
-    if (!options.names) {
-      return false
-    }
-    return options.names.includes(path.basename(route)) ||
+    return options.names?.includes(path.basename(route)) ||
       // match trailing slash
-      (options.names.includes('/') && route[route.length - 1] === '/')
+      (options.names?.includes('/') && route[route.length - 1] === '/') || false
```

Comment:
Nit: to prevent false from getting evaluated twice if `includes` evaluates to false

```suggestion
      ((options.names?.includes('/') && route[route.length - 1] === '/') ?? false)
```

Reply:
Why would it evaluate twice with `||` ?
https://jsfiddle.net/turnerran87/bdwoj5se/2/

Reply:
We only want the `false` that we added to be evaluated/returned if the _whole_ expression returns `undefined`. If the lefthand side evaluates to `false` as a result of the `includes` method or the `===` comparison, we don't need to evaluate/return our own `false`

```js
false || console.log('false') // output: false

// we'd want the following
false ?? console.log('false') // no output
undefined ?? console.log('false') // output: false
```

Reply:
Yeah good comment. Changed
WDYT?